x-pack/plugins/security_solution/public/common/components/events_viewer/index.tsx

```diff
@@ -185,6 +187,7 @@ const StatefulEventsViewerComponent: React.FC<Props> = ({
 
   const casesPermissions = useGetUserCasesPermissions();
   const CasesContext = cases.ui.getCasesContext();
+  const isLive = input.policy.kind === 'interval';
```

Comment:
is there a constant for 'interval' that we can use to track all the locations for this if it ever changes in the future?

Reply:
> is there a constant for 'interval' that we can use to track all the locations for this if it ever changes in the future?

There are 23 occurrences of the literal `'interval'` in 18 files in the `public` folder, and at a glance, many of them appear to refer to the same concept. In this diff, it's a lift and shift from this line in `mapStateToProps` (before it was deleted):

```ts
isLive: input.policy.kind === 'interval',
```

which may be viewed in context (even after this PR is merged) via the following [permalink](https://github.com/elastic/kibana/blob/4fe3cbbf22bba7edb864bc0e1371dce418d5edbc/x-pack/plugins/security_solution/public/common/components/events_viewer/index.tsx#L284).